src/TrackGenerator.h

```diff
@@ -79,6 +79,9 @@ class TrackGenerator {
   /** Filename for the *.tracks input / output file */
   std::string _tracks_filename;
 
+  /** Filename suffix for the *.tracks input / output file */
+  std::string _tracks_filename_suffix;
```

Comment:
Do you just have this so you can put in custom names for track files and run different geometries with the same parameters and not overwrite the tracks file?